app/views/shared/_search.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= form_tag(url, method: :get, remote: true, class: 'form-inline') do %>
+  <div class="form-group">
+    <div class="input-group">
+      <span class="input-group-addon" id="search-addon">
+        <span class="glyphicon glyphicon-search" aria-hidden="true"></span>
```

Comment:
We should use fontawesome here since we're using it everywhere else. `class="fa fa-search"` should work